### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by pierrz**
Just a simple translation of the various fields in French. The rendering has not been tested with the original layout because Tify is currently used with a tuned up 'skin' on our side.

### CODE REVIEW

File: dist/translations/de.json [hunk 1]
```diff
@@ -29,7 +29,6 @@
 	"Introduction": "Einführung",
 	"Key bindings": "Tastenbelegung",
 	"Jump to page": "Zu Seite springen",
-	"Key Bindings": "Tastenbelegung",
```
Comment (ipf) [inline dist/translations/de.json:32]: Why has this been removed?

File: dist/translations/fr.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
+{
```
Comment (ipf) [inline dist/translations/fr.json:1]: The translation file seems to work if it is moved to `static/translations/` instead of `dist/translations`.

**Review by ipf — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution.
Please have a look at the comments. When these issues are resolved, I'm happy to merge your Pull Request!

File: dist/translations/de.json (hunk 1 above)
Comment (t11r) [reply dist/translations/de.json:32]: This was an unused near-duplicate of the string two lines above, which was a bit elusive with the alphabetical order being broken 😉

There are probably more unused strings, maybe I will find the time to clean up both translations.

Comment (pierrz) [reply dist/translations/de.json:32]: Yes sorry, I noticed that I messed up with the alphabetic order once I commited ... I'll fix that with the next one :)

File: dist/translations/fr.json (hunk 2 above)
Comment (pierrz) [reply dist/translations/fr.json:1]: Strange, was working for me locally ... done

File: dist/translations/de.json (hunk 1 above)
Comment (t11r) [reply dist/translations/de.json:32]: That's not your fault, it was mixed up in the German translation, too.

File: dist/translations/fr.json (hunk 2 above)
Comment (t11r) [reply dist/translations/fr.json:1]: Files in `dist` are generated on release and should never be edited manually. We should clarify this in the readme.

Comment (pierrz) [reply dist/translations/fr.json:1]: Yes you're right ... dist is obviously meant to gather the 'end result'. I guess my last commit just was totally against those guidelines ! Let me know if you want a different commit or something.

**Review by ipf — APPROVED** [review]

**Pull request merged**